### PR TITLE
#170282790 fix the response data from trip stats

### PR DESCRIPTION
**Pull request by b0nbon1**
#### Description of Task to be completed?
- [x] allow manager to get count of his users trips
- [x] get managers users
- [x] allow users to get count of all of their trips
#### How should this be manually tested?
- git clone the repo
- run `npm install` to install dependencies
- run `npm run pretest-travis` and then `npm test` to run the tests
#### Any background context you want to provide?
- N/A?
#### What are the relevant pivotal tracker stories?
- [#170282790](https://www.pivotaltracker.com/n/projects/2407417/stories/170282790)

### CODE REVIEW

**Review by RaymondMwaura — DISMISSED** [review]
This seems to be in order. Fix the failing build first though so that it can be merged.

**Review by Mastel22 — DISMISSED** [review]
LGTM

**Review by nshaphan — DISMISSED** [review]
LGTM

**Review by gildniy — DISMISSED** [review]
Tested and working as expected.
Nice work @b0nbon1

File: src/controllers/hotels.controller.js [hunk 1]
```diff
@@ -231,6 +235,10 @@ class Hotel {
     }
 
     for (let i = 0; i < trips.length; i += 1) {
+      if (!trips[i].hotel) {
+        // eslint-disable-next-line no-continue
+        continue;
```
Comment (RaymondMwaura) [inline src/controllers/hotels.controller.js:240]: This is a useful function. I'll try it out in my features.

**Review by gildniy — APPROVED** [review]

**Review by RaymondMwaura — APPROVED** [review]
LGTM

**Review by nshaphan — APPROVED** [review]

**Pull request merged**